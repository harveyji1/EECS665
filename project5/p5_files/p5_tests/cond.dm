fn: () int{
    hello: int = 1;
    if(hello){
        give hello;
    }
}

main: () void{
    fn();
}