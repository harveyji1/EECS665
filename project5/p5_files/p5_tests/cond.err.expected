FATAL [3,8]-[3,13]: Non-bool expression used as a condition
Type Analysis Failed